ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -885,20 +886,33 @@ def test_create_deps_for_find_orphans(self, hostgroup1, host1,
 
         hostgroup1.retrieve()
 
-    @pytest.mark.skip(reason="Fails with 389-DS 1.4.0.22, see issue 7902")
     def test_find_orphan_automember_rules(self, hostgroup1):
         """ Remove hostgroup1, find and remove obsolete automember rules. """
         # Remove hostgroup1
 
         hostgroup1.ensure_missing()
 
         # Test rebuild (is failing)
+        # rebuild fails if 389-ds is older than 1.4.0.22 where unmembering
+        # feature was implemented: https://pagure.io/389-ds-base/issue/50077
+        if not have_ldap2:
+            raise unittest.SkipTest('server plugin not available')
+        ldap = ldap2(api)
+        ldap.connect()
+        rootdse = ldap.get_entry(DN(''), ['vendorVersion'])
+        version = rootdse.single_value.get('vendorVersion')
+        mo = re.search(r'(\d+)\.(\d+)\.(\d+)\.(\d+)*', version)
+        vendor_version = tuple(int(v) for v in mo.groups())
+        expected_failure = vendor_version < (1, 4, 0, 22)
```

Comment:
Would it be safer to use pkg_resources.parse_version instead of assuming that this will always be made up of ints?

Reply:
vendorVersion has the following format in the rootDSE: 389-Directory/1.3.8.4 B2019.037.1535

To use pkg_resources, I would need to trim the leading `389-Directory`/ and trailing `Byyyy.ddd.hhhh`, and in my mind it's not safer because there would also be an assumption on the format. But I can change if you prefer.

Reply:
The re already assumes a format of 4 ints.

I was mostly thinking it was a more python-ish way to compare version numbers.

I'm not super worried about this but if the format changes it could provoke a false positive failure I suppose.

Reply:
Ok, updated the PR with pkg_resource.parse_version to handle the comparison.